webapp/components/tabs.tsx

```diff
@@ -30,7 +30,7 @@ export const Tab = function ({
   return (
     <li
       className={`
-      box-border flex h-7 items-center rounded-md px-2 py-1 text-sm font-medium
+      box-border flex h-7 items-center rounded-md px-2 py-1 text-sm font-medium 
```

Comment:
Extra space added. Let's remove it so we don't add extra changes in the history 😅  (making a bit more annoying when using `git blame`)

```suggestion
      box-border flex h-7 items-center rounded-md px-2 py-1 text-sm font-medium
```

Reply:
Ops. fixed. thanks